MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,471 @@
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormCheckBoxItem]`Link`
+</td>
+<td>
+Generated for the Bool type property.
+[BoolDataTypeAttribute]
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormSwitchItem] `Link`
+</td>
+<td>
+Generated for the Bool type property.
+[BoolDataTypeAttribute]
+</td>
+</tr>
+
+<tr>
+<td>
+{{'[DataFormDateItem] `Link`
+</td>
+<td>
+Generated for the DateTime, DateOnly, DateTimeOffset type property.
+[DataType(DataType.Date)].
+[DataType(DataType.DateTime)].
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormTimeItem]
+`Link`
+</td>
+<td>
+Generated for the TimeSpan, TimeOnly  type property.
+[DataType(DataType.Time)].
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormPickerItem] `Link`
+</td>
+<td>
+Generated for the Enum type property.
+[EnumDataTypeAttribute]
+</td>
+</tr>
+
+<tr>
+<td>
+{{'[DataFormAutoCompleteItem]`Link`
+</td>
+<td>
+Generated for the Enum type property.
+[EnumDataTypeAttribute]
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormComboBoxItem]`Link`
+</td>
+<td>
+Generated for the Enum type property.
+[EnumDataTypeAttribute]
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormRadioGroupItem]`Link`
+</td>
+<td>
+Generated for the Enum type property.
+[EnumDataTypeAttribute]
+</td>
+</tr>
+</table>
+
+The following list of editors are supported:
+
+<table>
+<tr>
+<th>Editor</th>
+<th>Data Type/Attribute</th>
+<th>Input control loaded</th>
+</tr>
+<tr>
+<td>
+Text
+</td>
+<td>
+The String type property and any other type apart from the following specified cases.
+</td>
+<td>
+{{'[Entry]`Link`
+</td>
+</tr>
+<tr>
+<td>
+MultilineText
+</td>
+<td>
+The String type property with multi line text.
+[DataType(DataType.Multiline)] 
+</td>
+<td>
+{{'[Editor]`Link`
+</td>
+</tr>
+<tr>
+<td>
+Password
+</td>
+<td>
+The String type property with [DataType(DataType.Password)] attribute.
+</td>
+<td>
+{{'[Entry]`Link`
+</td>
+</tr>
+<tr>
+<td>
+CheckBox
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[CheckBox]`Link`
+</td>
+</tr>
+<tr>
+<td>
+Switch
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[Switch] `Link`
+</td>
+</tr>
+<tr>
+<td>
+Picker
+</td>
+<td>
+Enum and List type property. 
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[Picker]`Link`
+</td>
+</tr>
+<tr>
+<td>
+DatePicker
+</td>
+<td>
+The DateTime type property with date value.
+[DataType(DataType.Date)]
+[DataType(DataType.DateTime)]
+</td>
+<td>
+{{'[DatePicker]`Link`
+</td>
+</tr>
+<tr>
+<td>
+TimePicker
+</td>
+<td>
+Property with [DataType(DataType.Time)] attribute.
+</td>
+<td>
+{{'[TimePicker]`Link`
+</td>
+</tr>
+<tr>
+<td>
+ComboBox
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfComboBox]`Link`
+</td>
+</tr>
+<tr>
+<td>
+AutoComplete
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfAutoComplete]`Link`
+</td>
+</tr>
+
+<tr>
+<td>
+RadioGroup
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[RadioButton]`Link`
+</td>
+</tr>
+</table>
+
+## Layout options
```

Comment:
Remove this title and explain in Layout title

Reply:
Layout option removed